src/foam/core/reflow/FromCsvRefines.js

```diff
@@ -30,7 +30,10 @@ foam.CLASS({
       class: 'Function',
       name: 'fromCSV',
       value: function(str) {
-        return this.fromString(str?.replace(/,/g, ''));
+        if (typeof str === 'string' && str.includes(',')) {
```

Comment:
But in many countries (more than the other way around) they use , for . and . for ,. Maybe this should be an option, like what kind of delimiter we use.
See: https://support.ariba.com/item/view/176924

Reply:
It should be a parameter of the upload, just like setting the delimiter. Since it depends on who created the file, not on who uploaded it.